react/src/components/ResourcePresetSelect.tsx

```diff
@@ -164,10 +165,14 @@ const ResourcePresetSelect: React.FC<ResourcePresetSelectProps> = ({
               preset,
               disabled: disabled,
             };
-            // sort by disabled
-          }).sort((a, b) =>
-            a.disabled === b.disabled ? 0 : a.disabled ? 1 : -1,
-          ),
+          })
+            .sort(
+              (
+                a,
+                b, // by disabled
+              ) => (a.disabled === b.disabled ? 0 : a.disabled ? 1 : -1),
+            )
+            .sort((a, b) => localeCompare(a.value, b.value)), // by name
```

Comment:
I have placed the sort-related logic in the same location in the code and used localeCompare. Please check my commits.